workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinkerSystem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    [DisableAutoCreation]
+    public class EntityGameObjectLinkerSystem : ComponentSystem
```

Comment:
This shouldn't be a system

Reply:
The problem is that this doesn't really live anywhere. The init system we currently have is eventually just meant to be a default system that can be turned off. So the linker need to be accessible to systems somehow.